gslab_scons/release.py

```diff
@@ -97,10 +45,61 @@
     local_release = '/Users/%s/Google Drive/release/%s/' % (USER, name)
     local_release = local_release + version + '/'
     
-    release(vers              = version, 
-            DriveReleaseFiles = release_files,
-            local_release     = local_release, 
-            org               = organisation, 
-            repo              = repo,
-            target_commitish  = branch,
-            zip_release       = zip_release)
+    rtools.release(vers              = version, 
+                   DriveReleaseFiles = release_files,
+                   local_release     = local_release, 
+                   org               = organisation, 
+                   repo              = repo,
+                   target_commitish  = branch,
+                   zip_release       = zip_release)
+
+
+def inspect_repo():
+    '''Ensure the repo is ready for release.'''
+    if not rtools.up_to_date(mode = 'scons'):
+        raise ReleaseError('SCons targets not up to date.')
+    elif not rtools.up_to_date(mode = 'git'):
+        print "Warning: `scons` has run since your latest git commit.\n"
+        response = raw_input("Would you like to continue anyway? (y|n)\n")
+        if response in ['N', 'n']: 
+            sys.exit()
+
+
+def issue_size_warnings(file_MB_limit, total_MB_limit, bytes_in_MB):
+    '''Issue warnings if the files versioned in release are too large'''
+
+    # Compile a list of files in /release/ that are not versioned.
+    ignored = rtools.list_ignored_files()
+    release = rtools.create_size_dictionary('./release')
+    release = {k: release[k] for k in release.keys() if k not in ignored}
+    versioned = dict()
+
+    for file_name in release.keys():
+        versioned[file_name] = release[file_name]
```

Comment:
The creation of `versioned` seems unnecessary to me. It seems like it contains the exact same information as `release`.